Update conflict constraint for "nelmio/api-doc-bundle" in order to allow version 3 (#574)
sonata-project/SonataClassificationBundle: UPGRADE-3.x.md

UPGRADE 3.x
===========

UPGRADE FROM 3.x to 3.x
=======================

 ### Support for NelmioApiDocBundle > 3.6 is added

>  Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 was move to `Sonata\ClassificationBundle\Controller\Api\Legacy` and controllers for NelmioApiDocBundle v3 was added instead. If you extend them, you must change they using for correct Controller.

Reviewer: Did you have chance to see this suggestion?

>  Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 was move to `Sonata\ClassificationBundle\Controller\Api\Legacy` and controllers for NelmioApiDocBundle v3 was added instead. If you extend them, you must change they using for correct Controller.

Reviewer 2: ```suggestion
 Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved to `Sonata\ClassificationBundle\Controller\Api\Legacy` and controllers for NelmioApiDocBundle v3 was added instead. If you extend them, you must change they using for correct Controller.
```

Reviewer: I left the same suggestion before (https://github.com/sonata-project/SonataClassificationBundle/pull/574#discussion_r483854340).

>  Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 was move to `Sonata\ClassificationBundle\Controller\Api\Legacy` and controllers for NelmioApiDocBundle v3 was added instead. If you extend them, you must change they using for correct Controller.

Reviewer 2: ```suggestion
 Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 was move to `Sonata\ClassificationBundle\Controller\Api\Legacy` and controllers for NelmioApiDocBundle v3 was added instead. If you extend them, you must change and use the new controllers.
```
